public/doc/swagger-2.yaml

```diff
@@ -627,6 +704,15 @@ definitions:
         title: JSON Schema
         description: JSON Schema for the object
         readOnly: true
+responses:
+  InvalidEntity:
+    description: >-
+      The entity being created does not have either all the required parameters
+      specified or the parameter value is invalid
+  Forbidden:
+    description: The caller is forbidden to perform the action
+  Unauthorized:
+    description: The credentials are invalid when accessing the API externally.
```

Comment:
Would we want this to simply reference access not allowed? 
Possibly: The caller is not authorized to access this resource.